StressTest.ps1

```diff
@@ -0,0 +1,24 @@
+﻿[CmdletBinding()]
+param(
+    [string]
+    $BinSkimFolder = "bld\bin\x64_Release\netcoreapp3.1",
+
+    [string]
+    $OutputFolder = "c:\temp\",
+
+    [string]
+    $OutputFile = "binskim",
+
+    [string]
+    $InputPaths,
+
+    [int]
+    $Times = 1000
```

Comment:
>$Times [](http://example.com/codeflow?start=4&length=6)

Add a sentinel value that runs indefinitely? Or should people just set a super large value?

Reply:
Removed this, since i changed to a while loop.
Maybe we could add an argument like "stop when reach 100000".

This will prevent it from running forever

Reply:
What about parameter sets?